### DISCUSSION TITLE
Adding support for windows using cgofuse or dokany.

### DISCUSSION THREAD

**Discussion by shreyaspg** [rfjakob/gocryptfs#840] [Ideas & Feature requests]
I have been trying out and using gocryptfs on linux , and great work @rfjakob developing and maintaining this,  thanks.

Have you tried making gocryptfs work on windows using cgofuse(winfsp) or dokany. (We are aware of cppcryptfs, but having gocryptfs work on windows would be great)
Reading through past discussions, I see that the go-fuse that is used currently is not compatible with cgofuse.

Can you share some insights on how gocryptfs would work if it was enhanced to use the windows compatible fuse. I'm guessing the implementation in whole would need some changes apart from the api changes to make use of either windows fuse libraries.

**Comment by rfjakob** [comment]
Hi, no, and not on the agenda. Some technical reasons are explained here https://github.com/rfjakob/gocryptfs/issues/6#issuecomment-305417109 , but additionally I don't have the time to maintain a windows version. I'm glad cppcryptfs exists.
